contracts/lease/src/position/dto.rs

```diff
@@ -0,0 +1,29 @@
+use finance::liability::Liability;
+use serde::{Deserialize, Serialize};
```

Comment:
The order of the `use` statements is well defined: Rust standard, 3rd party crates, project crates, this crate (you see what the principle is?)

Reply:
this is not fixed

Reply:
formatting changes the positions of these lines